### DISCUSSION TITLE
Missing Functions in Library v3.0.0-rc1

### DISCUSSION THREAD

**Discussion by edivanteixeira** [elsa-workflows/elsa-core#4178] [Q&A] [answered]
I have created a project using libraries version 3.0.0-rc1, but some functions described in the v3 documentation are missing for me. I may be doing something wrong. What could it be? Thank you in advance.
<img width="700" alt="Screenshot 2023-06-30 at 12 43 40" src="https://github.com/elsa-workflows/elsa-core/assets/1633912/8860aca8-60af-4858-be42-be8d12d5587b">
<img width="665" alt="Screenshot 2023-06-30 at 12 45 27" src="https://github.com/elsa-workflows/elsa-core/assets/1633912/0a3bbbcd-7146-43aa-b460-bc21b1aab082">

**Comment by sfmskywalker** [comment]
I think that the documentation targets the preview packages, which are newer than the RC1. I’ll create an issue to keep a separate documentation deployment for the different releases.

**Reply by edivanteixeira** [reply]
Using RC1, how can I retrieve data from the previous activity without these methods?

**Reply by sfmskywalker** [reply, marked as answer]
Sure, in RC1 you need to first define a workflow variable (see the Variables tab in the designer).
Then, select the activity from which you want to access its output, and navigate to it Output tab.
From there, you need to associate the output with the variable you just created (just select the variable from the dropdown list).

This will cause the activity to store its output into the selected variable, which you can read from other activities.

**Reply by edivanteixeira** [reply]
Thank you very much, Sipke.
